application/models/Airports.php

```diff
@@ -3,13 +3,55 @@
 /**
  * Dummy model class that represents the airports our airline services
  */
-class Airports extends CSV_Model
+class Airports extends CI_Model
 {
+    private $baseId;
+    private $codesWeService;
 
     // Constructor
     public function __construct()
     {
         parent::__construct(APPPATH . DATA_AIRPORTS, 'id');
+        $this->baseId = 'YPR';
+        $this->codesWeService = ['YPR', 'ZMT', 'YZP', 'YXT'];
```

Comment:
We can get this information from http://wacky.jlparry.com/info/airlines/kite
So even less hard-coding.